packages/common/src/error.rs

```diff
@@ -301,6 +305,31 @@ pub enum ContractError {
 
     #[error("Verification Failed")]
     VerificationFailed {},
+
+    #[error("Channel doesn't exist: {id}")]
+    NoSuchChannel { id: String },
+
+    #[error("You can only send cw20 tokens that have been explicitly allowed by governance")]
+    NotOnAllowList,
+
+    #[error("Didn't send any funds")]
+    NoFunds {},
+    #[error("Only supports channel with ibc version ics20-1, got {version}")]
+    InvalidIbcVersion { version: String },
+    #[error("Only supports unordered channel")]
+    OnlyOrderedChannel {},
+    #[error("Got a submessage reply with unknown id: {id}")]
+    UnknownReplyId { id: u64 },
```

Comment:
nit: We already have an error `InvalidReplyId`. I do like that this one specifies the id so I would recommend deleting the old one and using this one instead.